docs/using-jda/interactions.md

```diff
@@ -39,12 +47,28 @@ To create commands you need to make some API requests. There are 2 types of comm
 - **Global**: These commands are available in every server your bot is in (regardless of sharding!) and direct message (Private Channels). These commands can take up to 1 hour to show up. _It is recommended to use guild commands for testing purposes._
 - **Guild**: These commands are only in the specific guild that you created them in and cannot be used in direct messages. These commands show up immediately after creation.
 
+### Creating Slash Commands
+
 You can create commands through these methods in JDA:
 
 - `updateCommands()`
 - `upsertCommand(name, description)`
 
-To create global commands you need to call these on a `JDA` instance and for guild commands on a `Guild` instance. Your bot needs the `applications.commands` scope in addition to the `bot` scope for your bot invite link. Example: <https://discord.com/oauth2/authorize?client_id=123456789&scope=bot+applications.commands> <- here
+!!! example
+    ```java
+    guild.updateCommands().addCommands(
+            Commands.slash("echo", "Repeats messages back to you.")
+                .addOption(OptionType.STRING, "message", "The message to repeat.")
+                .addOption(OptionType.INTEGER, "times", "The number of times to repeat the message.")
+                .addOption(OptionType.BOOLEAN, "ephemeral", "Whether or not the message should be sent as an ephemeral message.")
+    ).queue();
```

Comment:
This example should include another command which makes use of choices, since that isn't super intuitive by itself.